Fido2

arthurchan35 opened the pull request.

arthurchan35 committed:
LPS-92653 MFA FIDO2 web - server

arthurchan35 committed:
LPS-92653 MFA FIDO2 web - client, part I

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Commit:
LPS-92653 MFA FIDO2 web - client, part II

Commit:
LPS-92653 Add visibility controllers

for configuration to display only in instance settings and only when general MFA switch is enabled.

Commit:
LPS-92653 Add order with default high priority

Commit:
LPS-92653 SF Wordsmith

Commit:
LPS-92653 buildLang

Commit:
LPS-92653 SF

Commit:
LPS-92653 Better use of Optional

arthurchan35 committed:
cp

arthurchan35 committed:
sf
